src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,105 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double CAPACITY_MULTIPLIER = 1.5;
+    private T[] arrayList;
+    private int size = 0;
+
+    public ArrayList() {
+        arrayList = (T[]) new Object[DEFAULT_CAPACITY];
+    }
+
+    public void resizeArray() {
```

Comment:
```suggestion
    private void resizeArray() {
```
and put private ones after public ones.